package.json

```diff
@@ -62,6 +62,7 @@
     "lint": "eslint . --ext .ts --config .eslintrc",
     "postpack": "shx rm -f oclif.manifest.json",
     "posttest": "npm run lint",
+    "prepare": "npm run build",
```

Comment:
Remind me what the issue was here?

Reply:
When installing an npm module from git, `prepare` is run after cloning the repo. We need it here to compile the typescript into javascript in order to have something to import if we import the module from git.